src/routes/popup/earn/manage.tsx

```diff
@@ -126,6 +127,17 @@ export function ManageEarningsView() {
       const isAfterAllAOorPIYield =
         (updatedDelegationInfo[activeAddress] || 0) === 100 || (updatedDelegationInfo[PI_FLP_ID] || 0) === 100;
 
+      // track allocation update event
+      const changedTokens = tokens.filter((t) => t.flpId in changedDelegations);
+      const tokenIds = changedTokens.map((t) => t.processId);
+      const flpIds = changedTokens.map((t) => t.flpId);
+
+      await trackEvent(EventType.ALLOCATION_UPDATE, {
+        tokenNames: changedTokens.map((t) => t.Name).join(","),
+        tokenProcessIds: tokenIds.join(","),
+        tokenPercentages: flpIds.map((id) => `${changedDelegations[id]}%`).join(","),
+      });
+
```

Comment:
I think we should track analytics in its own try/catch or maybe at the end of this one so the user gets the "updated delegations" toast even if for whatever reason the analytics fails

Reply:
@7i7o should i remove await for trackEvent so it doesnot block the flow until it tracks the event ?

Reply:
That's an option, but then you wouldn't know if the event tracking throws any errors